File: Third semester/Synthesis and analysis of algorithms/Exercises/HufmanCoding/HufmanCoding/Node.cs
namespace HufmanCoding
{
    public class Node
    {
        public Node()
        {

        }

        public Node(int frequency)
        {
            this.Frequency = frequency;
        }

        public Node Left { get; set; }

        public Node Right { get; set; }

        public char Letter { get; set; }

        public int Frequency { get; set; }
    }
}

File: Third semester/Java/Exercises/Hufman/src/Node.java
public class Node {
    Node left;
    Node right;
    char letter;
    int frequency;

    public Node getLeft() {
        return left;
    }
    public void setLeft(Node left) {
        this.left = left;
    }
    public Node getRight() {
        return right;
    }
    public void setRight(Node right) {
        this.right = right;
    }
    public char getLetter() {
        return letter;
    }
    public void setLetter(char letter) {
        this.letter = letter;
    }
    public int getFrequency() {
        return frequency;
    }
    public void setFrequency(int frequency) {
        this.frequency = frequency;
    }
}
